static/css/components/cbox.less

```diff
@@ -134,7 +134,6 @@ div.coverFloat {
 div.floater {
   position: relative;
   font-family: @lucida_sans_serif-1;
-  width: 640px;
```

Comment:
This breaks the `manage covers` modal (or the other change in this file, not sure)

Reply:
Will investigate

Reply:
fixed issue by adding args to colorbox init, not by adding css